docs/src/man/basics.md

```diff
@@ -0,0 +1,2170 @@
+   2 │    85  female           3
+   3 │    98  female           2
+   4 │    95  transgender      2
+   5 │    78  female           2
+   6 │    89  male             1
+```
+
+set row 3 of columns `:Age`, `:Sex`, and `:Job` in-place:
+
+```jldoctest dataframe
+julia> df1[3, 1:3] = [78, "male", 4] 
+3-element Vector{Any}:
+ 78
+   "male"
+  4
+
+julia> df1
+6×3 DataFrame
+ Row │ Age    Sex          Job
+     │ Int64  String       Int64
+─────┼───────────────────────────
+   1 │    80  male             2
+   2 │    85  female           3
+   3 │    78  male             4
+   4 │    95  transgender      2
+   5 │    78  female           2
+   6 │    89  male             1
+```
+
+Now, let us explain how `DataFrameRow` can be used to mutate its parent data frame: 
+
+```jldoctest dataframe
+julia> dfr = df1[2, :] # DataFrameRow with the second row and all columns of df1
+DataFrameRow
+ Row │ Age    Sex     Job
+     │ Int64  String  Int64
+─────┼──────────────────────
+   2 │    85  female      3
+
+julia> dfr.Age = 98 # set value of col `:Age` in row `2` to `98` in-place
+98
+
+julia> dfr
+DataFrameRow
+ Row │ Age    Sex     Job
+     │ Int64  String  Int64
+─────┼──────────────────────
+   2 │    98  female      3
+
+julia> dfr[2:3] = ["male", 2] # set values of entries in columns `:Sex` and `:Job` 
+2-element Vector{Any}:
+  "male"
+ 2
+
+julia> dfr
+DataFrameRow
+ Row │ Age    Sex     Job
+     │ Int64  String  Int64
+─────┼──────────────────────
+   2 │    98  male        2
+```
+
+`DataFrameRow` is a view into a single row of a data frame.
+You can also create views into multiple rows of a data frame, which produce `SubDataFrame` objects.
+To learn more about `view` please have a look to [Views](@ref) section of this tutorial. 
+
+```jldoctest dataframe
+julia> sdf = view(df1, :, 2:3) # Column subsetting
+6×2 SubDataFrame
+ Row │ Sex          Job
+     │ String       Int64
+─────┼────────────────────
+   1 │ male             2
+   2 │ male             2
+   3 │ male             4
+   4 │ transgender      2
+   5 │ female           2
+   6 │ male             1
+
+julia> sdf[2, :Sex] = "female" # set value of col `:Sex` in second row to `female` in-place
+"female"
+
+julia> sdf
+6×2 SubDataFrame
+ Row │ Sex          Job
+     │ String       Int64
+─────┼────────────────────
+   1 │ male             2
+   2 │ female           2
+   3 │ male             4
+   4 │ transgender      2
+   5 │ female           2
+   6 │ male             1
+```
+
+Set value of multiple columns:
+
+```jldoctest dataframe
+julia> sdf[6, 1:2] = ["female", 3]
+2-element Vector{Any}:
+  "female"
+ 3
+
+julia> sdf
+6×2 SubDataFrame
+ Row │ Sex          Job
+     │ String       Int64
+─────┼────────────────────
+   1 │ male             2
+   2 │ female           2
+   3 │ male             4
+   4 │ transgender      2
+   5 │ female           2
+   6 │ female           3
+```
+
+In the above examples we have talked about `SubDataFrame`. In the given example,
```

Comment:
I do not understand the logic of this last example. Why it is here?

Reply:
Actually, one case had left so i thought i will include it here but now I have corrected it.